### PR TITLE
480259 - Calculator E2E Journey

### PR DESCRIPTION
**Pull request by evidentrilokkhandelwal**

### CODE REVIEW

File: src/EPR.Calculator.Service.Function/Services/CalculatorRunService.cs [hunk 1]
```diff
@@ -1,56 +1,177 @@
...
+        }
+
+        /// <summary>
+        /// Starts the calculator process.
+        /// </summary>
+        /// <param name="calculatorRunParameter">The parameters required to run the calculator.</param>
+        /// <returns>
+        /// A task that represents the asynchronous operation. The task result contains a boolean indicating success or failure.
+        /// </returns>
+        public async Task<bool> StartProcess(CalculatorRunParameter calculatorRunParameter)
```
Comment (tim-powell-eviden) [inline src/EPR.Calculator.Service.Function/Services/CalculatorRunService.cs:105]: This is a very long method - might be good to either break it into several smaller ones, or add some comments to highlight the start of different sections of it at a glance.

**Review by tim-powell-eviden — APPROVED** [review]

File: src/EPR.Calculator.Service.Function/Services/CalculatorRunService.cs (hunk 1 above)
Comment (evidentrilokkhandelwal) [reply src/EPR.Calculator.Service.Function/Services/CalculatorRunService.cs:111]: As suggested refactored the code

**Review by udaydefra — APPROVED** [review]
@evidentrilokkhandelwal  Thanks for making the changes

**Comment by evidentrilokkhandelwal** [comment]
Merge to main

**Pull request merged**